### ISSUE TITLE
Unable to access bigearthnet dataset

### ISSUE THREAD

**Issue by ankitpatnala** [tensorflow/datasets#3665]
I am not sure but i guess the url is not active anymore, so i am getting connection error

**Comment by kteedle** [comment]
> I am not sure but i guess the url is not active anymore, so i am getting connection error

Yes it's still the same. I think version 2 is out but we have to get the metadata and labels from the file and folder names which is confusing i think. I think v2.0 is revised BEN (reBEN) . v1.0 and v2.0 probably not in same format so the builder here cant be used just by changing the link perhaps

**Comment by kteedle** [comment]
updated BEN v1.0 link in the dataset builder file of bigearthnet. Waiting for approval for it to merge
